site/app/templates/admin/RainbowCustomization.twig

```diff
@@ -161,88 +161,106 @@
-                                    </div>
-                                    {{ bucket|capitalize }} ({{ customization_data[bucket]|length }} items)
-                                </li>
-                            {% endfor %}
-                        </ol>
+                <div style="width:45%;float:left;" id="buckets_used">
+                    <h3>Assigned Buckets</h3>
+                    <ol id="buckets_used_list" style="min-height:50px;background-color:var(--standard-beige);">
+                        {%  for bucket in used_buckets %}
+                            <li>
+                                <div class="bucket-percent" style="display: inline;">
+                                    <input aria-label="{{ bucket|capitalize }} percent" type="text" class="gradeable-custom-item-input" value="{{ bucket_percentages[bucket] }}" onblur="ClampPercent(this);" id="percent-{{ bucket }}"><span style="margin-right: 10px;">%</span>
+                                </div>
+                                {{ bucket|capitalize }} ({{ customization_data[bucket]|length }} items)
+                            </li>
+                        {% endfor %}
+                    </ol>
+                    Total: <span id="used_percentage">{% if 'used_percentage' in bucket_percentages|keys %}{{ bucket_percentages['used_percentage'] }}{% else %}0{% endif %}%</span>
+                </div>
+                <div style="width:45%;display:inline-block;margin-left:50px;" id="buckets_available">
+                    <h3>Available Buckets</h3>
+                    <ol id="buckets_available_list" style="min-height:50px;background-color:var(--standard-light-green);">
+                        {%  for bucket in available_buckets %}
+                            <li>
+                                <div class="bucket-percent" style="display: none;">
+                                    <input aria-label="{{ bucket|capitalize }} percent" type="text" class="gradeable-custom-item-input" value="0" onblur="ClampPercent(this);" id="percent-{{ bucket }}"><span style="margin-right: 10px;">%</span>
+                                </div>
+                                {{ bucket|capitalize }} ({{ customization_data[bucket]|length }} items)
+                            </li>
+                        {% endfor %}
+                    </ol>
+                </div>
+                <div id="gradeable_config" style="background-color:var(--standard-peachpuff);margin-top:50px;width:1000px;padding:5px;">
+                    <h2>Category/Gradeable Configuration:
+                        <i id="pencilIcon" class="fas fa-pencil-alt black-btn"></i>
+                    </h2>
+                    <div id="checkboxControls" style="display: none;">
+                        Extra credit:
+                        <input id="extra_credit_checkbox" type="checkbox" name="extra_credit"><br>
```

Comment:
The "Extra credit" text should be wrapped in a `<label>` HTML tag.  I believe the HTML validator would have caught this if it were run on this page.  It would be a good idea to add this route to the list of routes to check for accessibility in `site/cypress/e2e/Cypress-System/accessibility.spec.js`.

Reply:
I agree that this route should be listed in accessibility cypress test, and I am reluctant to make that specific edit in this PR, as it would open a can of worms. I opened up separate issue #10505 to properly address the issue.